### DISCUSSION TITLE
The password is invalid

### DISCUSSION THREAD

**Discussion by Gpan666** [LycheeOrg/Lychee#2754] [Q&A] [answered]
My password expired after updating from Lychee5 to Lychee6 and I was wondering if there was any way to change the password while keeping the data
![QQ20241128-020127](https://github.com/user-attachments/assets/96ad771d-da9b-47b9-8626-1b181d126d96)

**Comment by ildyria** [comment]
Updating from v5 to v6 shouldn't have changed your password. I don't know what happened there.

**Comment by ildyria** [comment, marked as answer]
I assume duplicate of https://github.com/LycheeOrg/Lychee/discussions/2755
